fastn-core/src/commands/build.rs

```diff
@@ -19,6 +21,47 @@ pub async fn build(
     {
         let documents = get_documents_for_current_package(config).await?;
 
+        if let Some(zip_url) = zip_url {
+            use sha2::digest::FixedOutput;
+            use sha2::Digest;
+
```

Comment:
Add `println!("Processing manifest.json ...")` somewhat same as for other files.